Pspcl/Pspcl.Models/LoginModel.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Pspcl.Models
+{
+    public class LoginModel
+    {
+        [Key]
```

Comment:
This is a model, not an entity. Wo dont specify relationships and constrainsts in models

Reply:
This is the separate LoginModel that I have to create in Pspcl.Models.

Reply:
You can create models, but my question is about constraints you are putting in a model

Reply:
Ok I will remove these.